plugin-server/tests/cdp/cdp-api.test.ts

```diff
@@ -241,6 +241,54 @@ describe('CDP API', () => {
             })
         })
 
+        it('doesnt include enriched values in the mock response', async () => {
+            hogFunction = await insertHogFunction({
+                ...HOG_EXAMPLES.simple_fetch,
+                ...HOG_INPUTS_EXAMPLES.simple_google_fetch,
+                ...HOG_FILTERS_EXAMPLES.no_filters,
+            })
+
+            const res = await supertest(app)
+                .post(`/api/projects/${hogFunction.team_id}/hog_functions/${hogFunction.id}/invocations`)
+                .send({ globals, mock_async_functions: true })
+
+            expect(res.status).toEqual(200)
+            expect(res.body).toMatchObject({
+                status: 'success',
+                error: null,
+                logs: [
+                    {
+                        level: 'debug',
+                        message: 'Executing function',
+                    },
+                    {
+                        level: 'debug',
+                        message: "Suspending function due to async function call 'fetch'. Payload: 2108 bytes",
+                    },
+                    {
+                        level: 'info',
+                        message: "Async function 'fetch' was mocked with arguments:",
+                    },
+                    {
+                        level: 'info',
+                        message: expect.not.stringContaining('developer-token'),
```

Comment:
How do you know this isn't just not set in the first place? there should ideally be some sort of assertion beforehand that proves that the value actually would be set as we are likely just testing nothing here.

Reply:
Added an additional test to make sure "real" requests include the `developer-token`.
[`049552a` (#26088)](https://github.com/PostHog/posthog/pull/26088/commits/049552a6c879cd8d888aa6aa55066192a591de04)

Reply:
Very nice